src/renderers/shaders/ShaderChunk/defaultnormal_vertex.glsl

```diff
@@ -1,7 +1,11 @@
 #ifdef FLIP_SIDED
 
-	objectNormal = -objectNormal;
```

Comment:
Leave `objectNormal` un-negated for future transforms (like displacement) that don't care about surface orientation.